UPGRADING-2.0.md

```diff
@@ -38,12 +38,93 @@ Upgrading From 1.x To 2.0
 
  * removed ``callback_filter`` configuration option for the jsonp_handler
 
- * ``exception_wrapper_handler`` is now the name of a service and not the name of a class
+ * ``exception_wrapper_handler`` config option was removed. Use normalizers instead.
+
+   Before:
+   ```yml
+   # config.yml
+
+   fos_rest:
+       view:
+           exception_wrapper_handler: My\exception_wrapper_handler
+   ```
+   ```php
+   namespace My;
+
+   class ExceptionWrapperHandler implements ExceptionWrapperHandlerInterface
+   {
+       public function wrap($data)
+       {
+           return new ExceptionWrapper(array('status_code' => 'foo'));
+       }
+   }
+   ```
+
+   After (if you use the Symfony serializer):
```

Comment:
I don't know if we should still advise to use the JMS serializer as it isn't updated a lot...